### DISCUSSION TITLE
Performing Arts Genres

### DISCUSSION THREAD

**Discussion by saumier** [culturecreates/artsdata-data-model#91] [Controlled Vocabularies]
Which controlled vocabulary should be used for performing arts genres?

**Comment by fjjulien** [comment]
Émilie, Dessa et moi avons eu une rencontre très intéressante avec Geneviève ce matin. Geneviève nous a parlé des travaux du Référentiel et de La Vitrine, notamment pour le développement de vocabulaires contrôlés décrivant les genres. La Vitrine va définir ses vocabulaires contrôlés avec les types DefinedTermSet et DefinedTerm de Schema plutôt qu'en (plutôt qu'en SKOS comme nous avons fait dans Artsdata). Ils vont ensuite saisir ces termes dans Event.keywords. Nous avons convenu de nous tenir informés de ces travaux car il serait avantageux tant pour Artsdata que La Vitrine et le Référentiel d'aligner leurs vocabulaires contrôlés. 

Nous avons aussi comparé l'approche de hiérachie classe - sous-classe, c'est-à-dire des relations un-à-un (comme le vocabulaire vocabulaire contrôlé d'événement d'Artsdata, lequel intègre un grand nombre de genres) vs un vocabulaire contrôlé dédié spécifiquement aux genres et permettant des relations un-à-plusieurs (ce qui permet d'avoir une plus grande granularité sans pour autant s'empêtrer dans une logique de hiérarchie). Par exemple,  le vocabulaire vocabulaire contrôlé d'événements comporte déjà un grand nombre de types qui désignent essentiellement des genres. Jusqu'où devrons-nous l'étendre pour répondre aux besoins des consommateurs de données? Sachant que l'information sur le genre peut être modélisée de façon indépendante de l'événement et être employée pour qualifier autant une oeuvre littéraire, qu'une oeuvre-spectacle ou qu'une représentation de spectacle (laquelle hérite son genre du spectacle lui-même), aurions-nous effectivement intérêt à doter Artsdata d'un vocabulaire de genre (et à alléger notre vocabulaire d'événements)? 

Je crois qu'Artsdata n'aura pas le choix d'adopter une approche de modélisation des genres lorsque nous allons commencer à charger des données d'oeuvres-spectacles. Lorsque ces données proviendront d'oeuvres-spectacles documentées dans Wikidata (notamment par Dorian), nous pourrons réutiliser les URIs de Wikidata pour désigner les genres. Lorsque les données proviendront de Scène Pro, nous utiliserons le vocabulaire contrôlé de Scène Pro. Mais Scène Pro et Wikidata n'ont pas été mis en concordance. Par ailleurs, Scène Pro est un vocabulaire unilingue français : il ne pourrait pas être utilisé dans d'autres langues. Et que ferons-nous des données d'oeuvres-spectacles en provenance de Scènes francophones ou d'autres sources? Ça va nous prendre un vocabulaire de référence multilingue pour faire les mises en concordance.
